### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by JINODK**
- Translate new strings
- Redo the entire translation to make it more natural and to use correct words

This is a big change that happen between translation string update so I don't know if every key is up-to-date, a quick visual check before merging is quite needed

### CODE REVIEW

**Comment by JINODK** [comment]
That should be all of it fixed for now

File: gui/public/i18n/vi/translation.ftl [hunk 1]
```diff
@@ -365,156 +365,156 @@ settings-osc-vrchat-network-trackers-elbows = Khuỷu tay
...
-onboarding-assign_trackers-back = Quay lại tra thông tin Wi-Fi
-onboarding-assign_trackers-title = Giao tracker
-onboarding-assign_trackers-description = Hãy chọn tracker nào nằm ở đâu. Chọn vị trí bạn muốn giao tracker cho
+onboarding-assign_trackers-back = Quay lại cài đặt Wi-Fi
+onboarding-assign_trackers-title = Gán tracker
+onboarding-assign_trackers-description = Chọn vị trí bạn muốn gán tracker bằng cách nhấn vào tên bộ phận muốn gán và chọn tracker
 # Look at translation of onboarding-connect_tracker-connected_trackers on how to use plurals
 # $assigned (Number) - Trackers that have been assigned a body part
 # $trackers (Number) - Trackers connected to the server
 onboarding-assign_trackers-assigned = { $assigned } of { $trackers ->
```
Comment (ImUrX) [inline gui/public/i18n/vi/translation.ftl:441]: there is an ``of`` in here, might have accidentally skipped it

**Review by ImUrX — CHANGES_REQUESTED** [review]
you can also remove all the ``[one]`` lines, I know vietnamese doesnt have singulars

File: gui/public/i18n/vi/translation.ftl (hunk 1 above)
Comment (JINODK) [reply gui/public/i18n/vi/translation.ftl:441]: I think you are right, been finding the missing `of` for a while but it might have been this

**Comment by JINODK** [comment]
I depend on those `[one]` lines to match up line count between the original English translation and my translation, just to make proofing easier, maybe I'll just leave it there and add some comments to clarify it

**Comment by JINODK** [comment]
Updated translation to address new strings and fixed merge conflict

File: gui/public/i18n/vi/translation.ftl [hunk 2]
```diff
@@ -121,260 +121,260 @@ tracker-table-column-type = Loại
...
 settings-serial-reboot = Khởi động lại
-settings-serial-factory_reset = Khôi phục cài đặt gối
+settings-serial-factory_reset = Khôi phục cài đặt gốc
 settings-serial-get_infos = Lấy thông tin
 settings-serial-serial_select = Chọn cổng Serial
 settings-serial-auto_dropdown_item = Tự động
 
 ## OSC router settings
-settings-osc-router = OSC router
+settings-osc-router = Router OSC
```
Comment (Louka3000) [inline gui/public/i18n/vi/translation.ftl:333]: Is the best term for "router" (as in, something that forwards data) in Vietnamese just router?

Comment (JINODK) [reply gui/public/i18n/vi/translation.ftl:333]: There's actually a word in Vietnamese for that, such as ***bộ chuyển tiếp*** for `repeater` and ***bộ định tuyến*** for `router`, however those words are closely tied to network ISPs and are rarely use outside of that, thus I just leave it as `router` there.

Comment (Louka3000) [reply gui/public/i18n/vi/translation.ftl:333]: Router in that context is indeed a router that forwards data through a (private) network, altho if the available terms are more specific, like for hardware, it's fine. As long as people understand :p

**Review by Louka3000 — APPROVED** [review]

File: gui/public/i18n/vi/translation.ftl (hunk 2 above)
Comment (JINODK) [reply gui/public/i18n/vi/translation.ftl:333]: Yip Vietnamese treats router as something that *magically split out internet* for some reason ¯\_(ツ)_/¯

**Review by ImUrX — APPROVED** [review]

**Comment by Louka3000** [comment]
Please rebase/merge your branch up to date

**Comment by JINODK** [comment]
Merged because rebase is a lil too dangerous for my liking

**Review by Louka3000 — APPROVED** [review]

**Pull request merged**